### ISSUE TITLE
Plugin causes crash in v2.0.0 release

### ISSUE THREAD

**Issue by bmish** [ember-template-lint/ember-template-lint#1150]
After upgrading to ember-template-lint 2.0.0, the template linter now crashes for me, when using either my custom template lint plugin or even a public plugin like `ember-template-lint-plugin-prettier`. When I comment out `plugins` in my template lint configuration, it no longer crashes.

Debugging into this a bit, I see that `configPath` is undefined. Looks likely to be related to #1138 (CC: @rwjblue @lifeart).

I have included simplified information about my setup and the call stack I was able to recover. My setup is a yarn workspace / monorepo. 

repository/workspace/myapp/.template-lintrc.js
```js
module.exports = {
  plugins: ['ember-template-lint-plugin-prettier']
};
```

Call stack from running ` repository/workspace/myapp yarn ember-template-lint .`:
```
internal/validators.js:125
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.dirname (path.js:1260:5)
    at resolvePlugin (repository/workspace/node_modules/ember-template-lint/lib/get-config.js:25:61)
    at processPlugins (repository/workspace/node_modules/ember-template-lint/lib/get-config.js:150:18)
    at module.exports (repository/workspace/node_modules/ember-template-lint/lib/get-config.js:317:5)
    at Linter.loadConfig (repository/workspace/node_modules/ember-template-lint/lib/linter.js:41:19)
    at new Linter (repository/workspace/node_modules/ember-template-lint/lib/linter.js:34:10)
    at run (repository/workspace/node_modules/ember-template-lint/bin/ember-template-lint.js:96:12)
    at Object.<anonymous> (repository/workspace/node_modules/ember-template-lint/bin/ember-template-lint.js:174:3)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
error Command failed with exit code 1.
```

**Comment by rwjblue** [comment]
Thanks for reporting @bmish! I'm a bit worried about the missing test coverage that this type of fundamental error seems to infer...

This is somewhat related to that comment I added here:

https://github.com/ember-template-lint/ember-template-lint/blob/4279eb30f9c7c20954951d26b74e1b34dcdb5b89/lib/get-config.js#L147-L150

At the moment, it is erroring because `options.configPath` is undefined (which is pretty common, AFAIK it is only going to be defined when `--config-path` is used). The temporary fix is to thread through the resolved project config path, but we do need to pass the _actual_ path to the file that included the string reference (this is what I was meaning in that comment).

I'm just stepping out for a bit, but will try to get it fixed once I'm back.

**Comment by rwjblue** [comment]
In order to repro the issue, I started a small refactor of our test harness in https://github.com/ember-template-lint/ember-template-lint/pull/1151.

**Comment by rwjblue** [comment]
Fix is included in [v2.0.1](https://github.com/ember-template-lint/ember-template-lint/releases/tag/v2.0.1).
